cmake/ExtendBuildTypes.cmake

```diff
@@ -26,6 +26,7 @@ set(SAN_FLAGS "-fsanitize=undefined -fsanitize=float-divide-by-zero -fno-sanitiz
 set(ASAN_FLAGS "-fsanitize=address")
 set(TSAN_FLAGS "-fsanitize=thread")
 set(STACK_FLAGS "-fstack-protector-all")
+set(DWARF_4_FAGS "-gdwarf-4")
```

Comment:
IIRC, 5 is the default dwarf version for gcc nowadays, but some of the dwarf-5 features are still problematic for some debuggers.

I'm curious to learn what prompted hardcoding this.  Discussion is nonessential.  :)

Reply:
Removing this, I'm also adding a better valgrind version.